src/sign.builds

```diff
@@ -10,8 +10,20 @@
 
   <UsingTask AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" TaskName="ReadSigningRequired" />
 
+  <ItemGroup>
+    <WindowsNativeLocation Include="$(NativeBinDir)\clrcompression.dll" />
+    <WindowsNativeLocation Include="$(NativeBinDir)_aot\clrcompression.dll" />
+  </ItemGroup>
+  
+  <Target Name="GenerateSignForWindowsNative">
+    <WriteSigningRequired AuthenticodeSig="$(AuthenticodeSig)"
+                          StrongNameSig="$(null)"
```

Comment:
Correct me if I'm wrong, but it's attempting to reference a variable with the name "null" which it can not find, so it nulls out. I didn't realize I could just leave it out and get the same effect though; I'll do that instead.